charter.md

```diff
@@ -121,18 +121,33 @@ Example:
   of no confidence
 
 The call for a vote of no confidence will happen in a public Steering Committee
-meeting and must be documented as a GitHub issue in the committee's
+meeting and must be documented as a GitHub issue ("the issue") in the committee's
 [repository][steering-repo].
 
 The call for a vote of no confidence must be made by a current member of the
-committee and must be seconded by another current member.
+committee ("the initiator") and must be seconded by another current member ("the seconder").
 
-The committee member who calls for the vote will prepare a statement which
-provides context on the reason for the vote. This statement must be seconded by
-the committee member who seconded the vote.
+The initiator will prepare a statement ("the charge") which provides
+context on the reason for the vote. The charge must be seconded by the seconder.
 
-Once a vote of no confidence has been called, the committee will notify the
-community through the following channels:
+Once the charge is known to the steering committee, the initiator may create the issue, and the target may prepare a
```

Comment:
if we are going to do this (which i am not in favor of!), then i'd prefer just a link. not copying the entire contents.

Reply:
I just don't see any valid grounds for treating charge & response disparately. The obvious motivations are not good (presumption of guilt or desire to hide the action from the community).

It's also extra hard to police the length if we just include a link, and it is easier for charge & response to be edited separately making one look silly etc.

Reply:
@lavalamp the charge is what the steering members vote on, so it should not be edited between the time it is posted and voted on. if it does need to change the clock has to restart as it is now a different charge.

It's up to the person who wants to rebut, to do what they want, where they want, the folks making the charge should not have any control over the voice of the person who is being charged. So no policing is needed.

thanks for your input and an honest try to make this better ... but i don't think it does.

Reply:
> it should not be edited between the time it is posted and voted on

The rules should probably state this if that is the intention! (the rule-change process doesn't state this and we're obviously not following it!)

> the folks making the charge should not have any control over the voice of the person who is being charged

Modulo that person needs to be able to have a voice, and it's likely the majority controls most or all of the official communication channels.

> So no policing is needed.

There's (at least) two categories of bad things that can happen here -- the board can attempt to stifle the voice of the minority (by e.g. surprising them, weird timing, dragging feet on making their response visible). But if we give the minority unconditional rights, then the minority can attempt to delay the process by being slow etc. Neither is acceptable.

You've got to think like a pen tester here :)

> if the motivations of a big set of individuals on the steering is suspect, that's what elections are for.

I 100% agree -- that's why my focus here is just on giving the community visibility into what is going on. Elections are the community's recourse if they don't like it. But they won't know to kick out a bad board if the board does its bad stuff without fanfare. (and likewise the community should be able to confidently reward a good board with additional terms in office)

Reply:
I’ve been on medical leave (and still am) but the use of the word target and charge here are not appropriate and do not approve such language. I’ve never seen anything like this in my oss career.

Reply:
The specific words aren't important to me, I've just swapped out target. I don't know a good substitute for charge but I'm happy to change it to something that doesn't sound, er, charged. The words I used apparently carry some connotations that I didn't intend, I'm sorry for using distracting words.

(Also sorry about whatever has you on medical leave & that you have to deal with this at the same time)